docs/en/observability/whats-new.asciidoc

```diff
@@ -3,21 +3,129 @@
 
 Here are the highlights of what's new and improved in {minor-version}.
 
-coming[7.13.0]
-
-// Other versions:
-// {observability-guide-all}/7.12/whats-new.html[7.12] |
-// {observability-guide-all}/7.11/whats-new.html[7.11] |
-// {observability-guide-all}/7.10/whats-new.html[7.10] |
-// https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
-// https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
-// https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
-// https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
-// https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
-// https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
+Other versions:
+{observability-guide-all}/7.13/whats-new.html[7.13] |
+{observability-guide-all}/7.12/whats-new.html[7.12] |
+{observability-guide-all}/7.11/whats-new.html[7.11] |
+{observability-guide-all}/7.10/whats-new.html[7.10] |
+https://www.elastic.co/blog/whats-new-elastic-observability-7-9-0-unified-agent-kpi-overview-dashboard[7.9] |
+https://www.elastic.co/blog/elastic-observability-7-8-0-released[7.8] |
+https://www.elastic.co/blog/elastic-observability-7-7-0-released[7.7] |
+https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
+https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
+https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
+
+[discrete]
+== Give {minor-version} a try
+
+Try {minor-version} now by deploying {es} and {kib} on
+{ess-product}[Elastic Cloud] or
+by https://www.elastic.co/start[downloading them].
 
 // tag::whats-new[]
+[discrete]
+== {agent}, {fleet}, and Integrations now generally available (GA)
+
+To make it easier to integrate your systems and applications with the {stack}, we are excited to announce
+a major improvement in the 7.14 release.
```

Comment:
This is weird to me. We say we're announcing **a major improvement**, and then discuss three different enhancements. I think we should pluralize this. Something like this:

```suggestion
three significant improvements in the 7.14 release.
```